GraphQL-README.md

```diff
@@ -313,4 +313,60 @@ query{
     }
   }
 }
-```
\ No newline at end of file
+```
+
+## New Endpoint: harvestData
+
+Allows you to fetch and filter harvest data for sustainable banks.
+
+### Query
+
+```graphql
+query {
+  harvestData(
+    tag: String!
+    customersServed: String
+    depositProducts: String
+    financialFeatures: String
+    services: String
+    institutionalInformation: String
+    policies: String
+    loanProducts: String
+    interestRates: String
+  ) {
+    customersServed
+    depositProducts
+    financialFeatures
+    services
+    institutionalInformation
+    policies
+    loanProducts
+    interestRates
+  }
+}
+```
+
+- *tag* is required and should be the tag of the bank you want to fetch data for.
```

Comment:
Unfortunately, this isn't quite what we need here. It needs to be possible to find all banks that serve "corporate" customers, without first having their tags